lib/person.rb

```diff
@@ -0,0 +1,22 @@
+require 'securerandom'
+
+class Person
+  attr_accessor :id, :name, :age, :parent_permission
+
```

Comment:
- [ ] [REQUIRED] Good job so far! I think it would be great if you made the `associations`(**:id**) to be `attr_reader` instead of `attr_accessor` and please remove  `:parent_permission` from `attr_accessor`💪🏻🔥💯

### [HINT] ✔️
```ruby
  # *:name and :age instance variables are the only ones that should be able to be modified that's why they are defined as attr_accessors (here both Getters and Setters are defined)
  attr_accessor :name, :age 
  # *:id should only be able to be retrieved so only the Getter is defined
  attr_reader :id 
  ```

Reply:
@mukhammadrizooff Ok got it.